src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard/MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard key={movie.imdbId} {...movie} />
```

Comment:
don't use spread for the pass a props
also, put the whole movie to the `MovieCard`
```suggestion
      <MovieCard key={movie.imdbId} movie={movie} />
```